### DISCUSSION TITLE
Xilinx encoder support

### DISCUSSION THREAD

**Discussion by fairbairn** [OvenMediaLabs/OvenMediaEngine#1217] [Ideas] [answered]
A good bit of our infrastructure rides on AWS.

AWS specifically built an ARM-based video encoding virtual machine called the VT1, which uses an on-board Xilinx GPU card for video encoding operations.

The cost-benefit of this hardware is significant.

Are you interested in supporting the Xilinx video encoder?

Information can be found here..

https://xilinx.github.io/video-sdk/v1.5/getting_started_on_vt1.html

**Comment by Keukhan** [comment, marked as answer]
@fairbairn 

I recently heard that AWS is offering VT1 instances with GPU support. I expect to be able to give customers a cost advantage using OME. 
This feature will be planned as a high priority. I will give you good news soon. Please look forward to it.

Thanks.

**Reply by tonywingmah** [reply]
Wondering what the progress on this is? I just saw this:
https://www.wowza.com/docs/introduction-to-amd-xilinx-acceleration-with-wowza-streaming-engine-transcoding-flow

**Reply by fairbairn** [reply]
It's very much underway.  Commits already in progress.

**Reply by tonywingmah** [reply]
Any plans for a multiviewer?

https://www.youtube.com/watch?v=N2QNug0D-Tg&t

**Reply by tonywingmah** [reply]
any updates on this? MA35D is now rentable on Azure
